go-app/main.go

```diff
@@ -106,46 +102,73 @@ func homeHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	switch page {
+	case "loglevel":
+		loadLogLevel(w, r)
 	case "UploadConfig":
-		configs, err := settings.UploadConfigFile(r, project_id, cloudConfigs)
-		getFeedBack(err, "Upload Config")
-		if err == nil {
-			cloudConfigs = configs
-		}
-		cloudConfigs = configs
-		feedbackTempl.Execute(w, feedBack)
+		loadUploadConfig(w, r)
 	case "editConfig":
-		if r.FormValue("action") == "Save" {
-			err := settings.SaveConfig(r, bucket_edit, cfg_edit)
-			getFeedBack(err, "Edit Config")
-			feedbackTempl.Execute(w, feedBack)
-		} else {
-			bck, cfg, content, err := settings.DisplayConfig(w, r, project_id, region_id)
-			bucket_edit = bck
-			cfg_edit = cfg
-			if err != nil {
-				getFeedBack(err, "Edit Config")
-				feedbackTempl.Execute(w, feedBack)
-				return
-			}
-			edit_configTempl.Execute(w, content)
-		}
+		loadEditConfig(w, r)
 
 	case "deleteConfig":
-		configs, err := settings.DeleteConfig(r, project_id, region_id, cloudConfigs)
-		getFeedBack(err, "Delete Config")
-		if err == nil {
-			cloudConfigs = configs
-		}
-		feedbackTempl.Execute(w, feedBack)
+		loadDeleteConfig(w, r)
 	default:
-		err := report.AnalyseLog(w, r, project_id, region_id)
+		loadAnalyseLog(w, r)
+	}
+
+}
+func fillUploadCfgPage(w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: The other functions for opening a different page start with load*, can we follow the same naming convention?

Reply:
Yes sure